common/models/user.js

```
@@ -1,12 +1,13 @@
 "use strict";
 const UserService = require("../services/user.service");
+const logger = require("../utils/logger.js");
 
 module.exports = function (user) {
   user.updateById = async (id, name, dob, address, description) => {
     try {
       return await UserService.updateById(id, name, dob, address, description);
     } catch (err) {
-      return Promise.reject(err);
+      logger.error(err);
```

well...
This is ignoring the error thrown. @dd201702